core/src/scss/components/main-nav/_main-nav.scss

```diff
@@ -42,6 +42,8 @@
 //
 
 .su-main-nav {
+  $_su-main-nav-color-map: $su-nav-colors;
```

Comment:
```suggestion
```
Looks like we can get rid of this line because `$_su-main-nav-color-map` isn't used anywhere.